src/pages/view/FilterBar/SavedFilterActions.js

```diff
@@ -0,0 +1,176 @@
+            {
+                show: showFilterAction,
+                label: i18n.t('Rename'),
+                action: () => Promise.resolve(openDialogForRename()),
+            },
+            {
+                show: showFilterAction,
+                label: i18n.t('Delete'),
+                action: () => doDeleteFilter(currentUser),
+            },
+            {
+                show: savedFilterHasChanges && showFilterAction,
+                label: i18n.t('Save'),
+                action: () => doSaveFilter(activeFilter),
+                dialogAction: 'update',
+            },
+            {
+                show: savedFilterHasChanges,
+                label: i18n.t('Save as new filter'),
+                action: openDialogForNewFilter,
+                skipCheck: true,
+            },
+            {
+                show: showFilterAction,
+                label:
+                    activeFilter?.visibility === privateVisiblity
+                        ? i18n.t('Make it public')
+                        : i18n.t('Make it private'),
+                action: () => doToggleFilterVisibility(currentUser),
+                dialogAction:
+                    activeFilter?.visibility === privateVisiblity
+                        ? i18n.t('make public')
+                        : i18n.t('make private'),
+            },
+        ]
+
+        return (
+            <FlyoutMenu>
+                {actions.map(
+                    ({ show, label, action, skipCheck, dialogAction }) =>
+                        show && (
+                            <MenuItem
+                                key={label}
+                                dense
+                                label={label}
+                                onClick={handleAction({
+                                    action,
+                                    label,
+                                    skipCheck,
+                                    dialogAction,
+                                })}
+                            />
+                        )
+                )}
+            </FlyoutMenu>
+        )
+    }, [
+        savedFilterHasChanges,
+        showFilterAction,
+        activeFilter,
+        currentUser,
+        doDeleteFilter,
+        doSaveFilter,
+        doToggleFilterVisibility,
+        handleAction,
+        openDialogForNewFilter,
+        openDialogForRename,
+    ])
+
+    return (
+        hasActiveSavedFilter &&
+        (showFilterAction || savedFilterHasChanges) && (
+            <>
+                <DropdownButton
+                    dataTest="more-actions-button"
+                    secondary
+                    small
+                    showArrow={false}
+                    open={moreOptionsIsOpen}
+                    disabledWhenOffline
+                    onClick={toggleMoreActions}
+                    icon={<IconMore16 color={colors.grey700} />}
+                    component={savedFilterActions}
+                />
```

Comment:
Children is required for `<DropdownButton />`. can we pass an empty fragment to avoid the error? 

```ts
<DropdownButton ...>
   <></>
</DropdownButton 
```

I know the ideal solution should be to make the prop optional, but let's try to avoid modifying validations for an existing component.

![image](https://github.com/user-attachments/assets/1646e2c5-68b1-4217-8a2e-3876e8259239)

Also add an space between dropdown and `<ConfirmActionDialog`